assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -4319,7 +4337,7 @@ private String getReviewReport(ContentResource cr, String assignmentReference) {
                 return contentReviewService.getReviewReportStudent(contentId, assignmentReference, userDirectoryService.getCurrentUser().getId());
             }
         } catch (Exception e) {
-            log.warn(":getReviewReport(ContentResource) {}", e.getMessage());
+            log.debug(":getReviewReport(ContentResource) {}", e.getMessage());
```

Comment:
Note to reviewers: I've done this change as this method is invoked pretty frequently for content review items that are in 'Error' status (e.g. student submission is less than 20 words, etc.). It contributes to log bloat with lines like:
`:getReviewReport(ContentResource) Report status: 2`